.NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/HolidayParserJap.cs

```diff
@@ -0,0 +1,414 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+using Microsoft.Recognizers.Definitions.Japanese;
+using Microsoft.Recognizers.Text.Number;
+using Microsoft.Recognizers.Text.Number.Japanese;
+
+using DateObject = System.DateTime;
+
+namespace Microsoft.Recognizers.Text.DateTime.Japanese
+{
+    public class HolidayParserJap : IDateTimeParser
+    {
+        public static readonly string ParserName = Constants.SYS_DATETIME_DATE; //"Date";
+
+        private static readonly IExtractor IntegerExtractor = new IntegerExtractor();
+
+        private static readonly IParser IntegerParser = new BaseCJKNumberParser(new JapaneseNumberParserConfiguration());
+
+        public static readonly Dictionary<string, Func<int, DateObject>> FixedHolidaysDict = new Dictionary<string, Func<int, DateObject>>
+        {
+
+            #region Fixed Date Holidays
+
+            {"元旦", NewYear},

```

Comment:
Ideally such lists and dictionaries should be moved to the target language YAML file.

Reply:
Sorry for this late response. About this modification, There was a mistake from my part. In this file I should have removed all the holidays, because they are related to the Chinese calendar and have nothing to do with Japanese. I'll make a new pool request to fix this.